assets/i18n/zh-hant.json

```diff
@@ -26,7 +26,7 @@
 	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_3": "如果您想要加入支持這些成果的數百名人員，請了解更多有關<a href=\"/donate\">捐贈 Oppia 基金會</a>或<a href=\"/contact\">以其它方式參與</a>。",
 	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_4": "基金會的董事為 Ben Henning、Jacob Davis、與 Sean Lip。有關基金會的<a href=\"https://drive.google.com/folderview?id=0B9KSjiibL_WDfjlpUUpqampZSTBOTmRqM2IyaGN3R3NlVWstZlhKcjlZQ3VMTHJBQWNGTFU&usp=sharing\" class=\"inline-links\">章程與記錄</a>可供讀取。如果您想要聯絡基金會，請寄送電子郵件到：<a href=\"mailto:admin@oppia.org\" class=\"inline-links\">admin@oppia.org</a>。",
 	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_4_HEADING": "董事",
-	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5": "所有在 Oppia 裡的探索內容皆依據<a href=\"https://creativecommons.org/licenses/by-sa/4.0/legalcode\" class=\"inline-links\">姓名標示-相同方式分享 4.0 </a>許可協議及捨棄歸屬要求（按類別第3(a)(1)節與第 3(a)(2) 節）。如果您要重複使用來自此網站的內容，雖不強制但我們鼓勵您補充加入相關探索頁面的連結。",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5": "所有在 Oppia 裡的探索內容皆依據<a href=\"https://creativecommons.org/licenses/by-sa/4.0/legalcode.zh-Hant\" class=\"inline-links\">姓名標示-相同方式分享 4.0 </a>許可協議。",
```

Comment:
Hi @seanlip, This PR is failing the backend test for the changes in the link, though the new link seems more correct as it shows the license text in zn-hant language.

Reply:
The failing test description:

```
def test_html_in_translations_is_preserved_correctly(self):
        """Tests that HTML in translated strings matches the original
        structure.
        """
.....
```

Reply:
Possible solutions:

1. Change the URL to the expected format.
2. Change the test function so that it allows such a link.

I would prefer the 1st one for now as it's simple and the 2nd solution can be complicated and can lead to security issues (if we miss anything by chance).

Reply:
As per offline discussion with @seanlip, we finalized to go with the 1st solution for now (i.e, Change the URL to the expected format.).